frontend/apps/api_docs/v1/tests/test_docs.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+import pytest
+from bs4 import BeautifulSoup
+from response_builder.v1.models.base import RootModel
+
+
+def extract_code_block(html, h4_id):
```

Comment:
Untested but I think you might be able to get all the code blocks you want (e.g ignoring the elections endpoint) with:

```
soup.find("h2", class_="postcode-search-2").find_parent("div").find_all("div", class_="api-action-body")
```
This might end up being a better way to ensure that future code blocks are included.

Reply:
Thanks. I've pushed fce437223d91b57f0130c9e82555068e725e2a02 which incorporates that suggestion.